pageserver/src/storage_sync2.rs

```diff
@@ -657,10 +716,6 @@ impl RemoteTimelineClient {
         // from latest_files, but not yet scheduled for deletion. Use a closure
         // to syntactically forbid ? or bail! calls here.
         let no_bail_here = || {
-            for name in names {
```

Comment:
Did we lose this part?

Reply:
No, it moved upwards in this function when I opened the PR. Moved it back down here.